### PR TITLE
[chromecast] Updated and added complete german translation

### PR DESCRIPTION
**Pull request by cweitkamp**
- Updated and added complete german translation
- Clean-up of xml-files

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

**Comment by martinvw** [comment]
@CWempe @cweitkamp keeps you busy ;-)

**Comment by cweitkamp** [comment]
These are very similar to the Kodi translations. Easy peasy going. ;-)

File: addons/binding/org.openhab.binding.chromecast/ESH-INF/i18n/chromecast_de.properties [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# binding
+binding.chromecast.name = Chromecast Binding
+binding.chromecast.description = Dieses Binding integriert Chromecast Ger�te (z.B. Chromecast, Chromecast Audio oder Chromecast Ultra).
+
+# thing types
+thing-type.chromecast.audiogroup.label = Chromecast Audiogruppe
+thing-type.chromecast.audiogroup.description = Audiogruppe von mehreren Chromecast Audio oder Media Playern.
```
Comment (CWempe) [inline addons/binding/org.openhab.binding.chromecast/ESH-INF/i18n/chromecast_de.properties]: Or: "Audiogruppe mehrerer Chromecast Audio oder Media Player"
But I am not sure which sounds better.

Comment (cweitkamp) [reply addons/binding/org.openhab.binding.chromecast/ESH-INF/i18n/chromecast_de.properties]: @CWempe Tahnks. I am gonna change it to: "Audiogruppe aus mehreren Chromecast Audio oder Media Playern". wdyt?

**Comment by martinvw** [comment]
Thanks!

**Pull request merged**